test/broker_test/broker_test.go

```diff
@@ -423,6 +515,29 @@ var _ = test.DescribeTestsFor(test.TestCase{
 							}, "services.0.plans.0.schemas", "{invalid")
 						})
 					})
+
+					Context("when tls credentials are not base64 encoded", func() {
+						It("returns 400", func() {
+							ctx.SMWithOAuth.POST(web.ServiceBrokersURL).WithJSON(postBrokerRequestWithInvalidBase64TlsEncoding).
+								Expect().
+								Status(http.StatusBadRequest).
+								JSON().Object().Keys().Contains("error", "description")
+
+							assertInvocationCount(brokerServer.CatalogEndpointRequests, 0)
+						})
+					})
+
+					Context("when tls credentials are invalid", func() {
+						It("returns 400", func() {
+							reply := ctx.SMWithOAuth.POST(web.ServiceBrokersURL).WithJSON(postBrokerRequestWithTls).
+								Expect().
+								Status(http.StatusCreated).
+								JSON().Object()
+							reply.ContainsMap(expectedBrokerResponseTls)
+
+							assertInvocationCount(brokerServer.CatalogEndpointRequests, 1)
+						})
+					})
```

Comment:
test all possible combinations of basic and/or tls - 4

Reply:
Done!